README.md

```diff
@@ -1,24 +1,139 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [📹 video presentation](#video-presentation)
+  - [:camera: screenshots](#screenshots)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] [OPTIONAL] Kindly remove the Live Demo from the table of content since you do not have it in the body of the README and this project has not been deployed. This will help maintain a clean code.

Reply:
fixed